tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -67,23 +67,23 @@ public sealed class OrderEntity : EntityBase
 
         public static async Task<string> FetchOrderDescriptionFromOrderId(string connectionString, string orderId)
         {
-            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
+            return (await SqlRunner.QueryFirstAsync<string>(connectionString, @"SELECT
                          [Description]
                          FROM [Order]
                          WHERE [OrderId] = @orderId", new { orderId }));
         }
 
         public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
         {
-            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
+            return (await SqlRunner.QueryFirstAsync<string>(connectionString, @"SELECT
                          [LastUpdatedByName]
                          FROM [Order]
                          WHERE [OrderId] = @orderId", new { orderId }));
         }
 
         public static async Task<OrderEntity> FetchOrderByOrderId(string connectionString, string orderId)
         {
-            return (await SqlRunner.QueryFirstAsync<OrderEntity>(connectionString, $@"SELECT
+            return (await SqlRunner.QueryFirstAsync<OrderEntity>(connectionString, @"SELECT

```

Comment:
One last request (I promise) – can you do a little SQL tidying? Remove square brackets where not required, add in schema identifiers (`dbo`) where not present and add terminating semicolons?

Reply:
👍